ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureCloudDestinationSync.java

```diff
@@ -693,18 +704,20 @@ public FindResult findEntriesSince(String partitionPath, FindToken findToken, lo
    * @param blobContainerClient BlobContainer client
    * @return The number of blobs erased
    */
-  protected int eraseBlobs(List<BlobItem> blobItemList, BlobContainerClient blobContainerClient) {
+  protected int eraseBlobs(List<BlobItem> blobItemList, BlobContainerClient blobContainerClient, Set<Pair<Short, Short>> deletedContainers) {
     int numBlobsPurged = 0;
     long now = System.currentTimeMillis();
     long gracePeriod = TimeUnit.DAYS.toMillis(cloudConfig.cloudCompactionGracePeriodDays);
     for (BlobItem blobItem: blobItemList) {
       if (shutdownCompaction.get()) {
-        logger.info("Shut down compaction for container {}", blobContainerClient.getBlobContainerName());
+        logger.info("[COMPACT][2] Shut down compaction for partition {}", blobContainerClient.getBlobContainerName());
         break;
       }
       Map<String, String> metadata = blobItem.getMetadata();
+      Pair<Short, Short> shortShortPair = new Pair<>(Short.parseShort(metadata.get(CloudBlobMetadata.FIELD_ACCOUNT_ID)),
```

Comment:
naming it with the usage instead of the datatype. Probably shortShortPair -> accountContainerIds?

Reply:
doen thank you